includes/lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -213,6 +207,32 @@ public function send_event( $event, $properties = array(), $event_timestamp = nu
 		// to ever add event tracking at the user level.
 		$properties['_ui'] = str_replace( 'www.', '', wp_parse_url( site_url(), PHP_URL_HOST ) );
 		$properties['_ul'] = $user->user_login;
+
+		return $this->send_anonymous_event( $event, $properties, $event_timestamp );
+
+	}
+
+
+	/**
+	 * Send an event to Tracks.
+	 *
+	 * @param string   $event           The event name. The prefix string will be
+	 *                                  automatically prepended to this, so please supply this string without a
+	 *                                  prefix.
+	 * @param array    $properties      Event Properties.
+	 * @param null|int $event_timestamp When the event occurred.
+	 *
+	 * @return bool
+	 **/
+	public function send_anonymous_event( $event, $properties = array(), $event_timestamp = null ) {
```

Comment:
For the normal events, sounds a little weird to me going through the `send_anonymous_event`. Maybe we could call it as `create_pixel`, or something like that, and also have `send_anonymous_event` to be called in the `save_exit_survey`. The `send_anonymous_event` would also call the `create_pixel` (and it wouldn't do anything else for now). Does it make sense to you?

Reply:
Changed this up in bda3c37, as the anonymous method has it's own logic now.